pkg/server/server.go

```diff
@@ -124,12 +131,26 @@ func WithListObjectsMaxResults(limit uint32) OpenFGAServiceV1Option {
 	}
 }
 
+// WithMaxConcurrentReadsForListObjects sets a limit on the number of datastore reads that can be in flight for a given Check call.
+// This number should be set depending on the RPS expected for Check and ListObjects APIs, the number of OpenFGA replicas running,
+// and the number of connections the datastore allows.
+// E.g. if Datastore.MaxOpenConns = 100 and assuming that each ListObjects call takes 1 second and no traffic to Check API:
+// - 1 OpenFGA replica and expected traffic of 100 RPS => set it to 1.
+// - 1 OpenFGA replica and expected traffic of 1 RPS => set it to 100.
+// - 2 OpenFGA replicas and expected traffic of 1 RPS => set it to 50.
 func WithMaxConcurrentReadsForListObjects(max uint32) OpenFGAServiceV1Option {
 	return func(s *Server) {
 		s.maxConcurrentReadsForListObjects = max
 	}
 }
 
+// WithMaxConcurrentReadsForCheck sets a limit on the number of datastore reads that can be in flight for a given Check call.
+// This number should be set depending on the RPS expected for Check and ListObjects APIs, the number of OpenFGA replicas running,
```

Comment:
This is just for Check, not ListObjects.

Reply:
Right, but it's important to account for traffic for both when setting these values